src/documents/ModelRelay.md

```diff
@@ -93,9 +128,22 @@ This style is used when we require a [Model Transformation](to-do/ModelTransform
         <tr>
             <td>transform</td>
             <td>JSON (full <a href="to-do/ModelTransformation.md">Model Transformation</a> document)</td>
-            <td>A long form which allows any valid Model Transformation document to be used to mediate the relay</td>
+            <td>A long form of <code>singleTransform</code> which allows any valid Model Transformation document to be used to mediate the relay</td>
             <td>See this <a href="http://wiki.gpii.net/index.php/Architecture_-_Available_transformation_functions">list of available transformation functions</a> for more information.</td>
         </tr>
+        <tr>
+            <td><code>forward</code> (optional)</td>
+            <td>One of the strings <code>never</code>, <code>initOnly</code>, <code>liveOnly</code> or <code>always</code> (the default)</td>
+            <td>Control the situations in which the forward leg of the transform operates - <code>initOnly</code> will have it only operated during the <a href="#the-initial-transaction">initial transaction</a> that this 
```

Comment:
This should include a link to the "Controlling Propagation Through a Relay Rule" section, which explains forward and backward.